packages/framework/src/Model/Seo/SeoSettingFacade.php

```diff
@@ -59,37 +54,55 @@ public function getDescriptionsMainPageIndexedByDomainIds(array $domainConfigs)
 
     /**
      * @param int $domainId
-     * @return string
+     * @return string|null
      */
-    public function getTitleAddOn($domainId)
+    public function getTitleAddOn(int $domainId): ?string
```

Comment:
adding return type and typehint is a BC-break. please remove it.

Reply:
Removed.